### ISSUE TITLE
Why not expose all the parameters of the RandomForestRegressor?

### ISSUE THREAD

**Issue by jonathanng** [manifoldai/merf#23]
Also, would using the ExtraTreesRegressor also in this framework? It seems simple to just add a base_estimator parameter, allowing the user to use whatever he/she wants?

I can submit a pull request if it makes sense to add these features.

**Comment by arose13** [comment]
I just added this also. I'm thinking of just permanently forking this problem and seeing how bad complete model generalizations would be.

**Comment by arose13** [comment]
Check out https://github.com/arose13/GREM since it is the soltion to your problem

**Comment by resdntalien** [comment]
This is really nice! I've been meaning to do this for awhile myself because it is generalizable like you propose. I wonder if we can just merge these into a single project. 

I also have another idea make the fit() call look more like the fit calls of PyTorch and Keras where you pass a validation set as well. This is the right way to check if you've run enough iterations, etc.
